src/test/scala/org/kibanaLoadTest/test/IngestionTest.scala

```diff
@@ -59,19 +91,43 @@ class IngestionTest {
     )
 
     val esClient = new ESClient(esConfig)
-    val simLogFilePath = getClass.getResource("test/simulation.log").getPath
-    val lastRunFilePath = getClass.getResource("test/lastRun.txt").getPath
+    val simLogFilePath = getClass.getResource("/test/simulation.log").getPath
+    val lastRunFilePath = getClass.getResource("/test/lastRun.txt").getPath
+    val responseFilePath = getClass.getResource("/test/response.log").getPath
     val metaJson = Helper.getMetaJson(lastRunFilePath, simLogFilePath)
-    val requests = LogParser.getRequests(simLogFilePath)
+    val (requestsTimeline, concurrentUsers) =
+      LogParser.parseSimulationLog(simLogFilePath)
+    val responses = ResponseParser.getRequests(responseFilePath)
+    for (i <- 0 to responses.length - 1) {
+      val value = responses(i)
+      responses(i) = value.copy(
```

Comment:
I get that this is just a test, but for some reason my eyebrow raised on this one.
Unless, `value.copy(...)` is some sort of merge functionality.

In my JS brain, it looks like:
``` 
let xs = ...;
const x = xs[i];
xs[i] = x.copy(...);  // Imaginary function application.

// If the above is correct, should it be:
responses[i] = responses[i].copy(...) // does not look like value is used anywhere else.
```

Reply:
Fixed, commit is coming. Good point!